integration/test-integration/pom.xml

```
@@ -315,7 +315,7 @@
                 </os>
             </activation>
             <properties>
-                <apictl.package>apictl-${apictl.version}-macosx-x64.tar.gz</apictl.package> <!-- Use same amd64 distribution until arm64 available -->
+                <apictl.package>apictl-${apictl.version}-darwin-arm64.tar.gz</apictl.package> <!-- Use same amd64 distribution until arm64 available -->
```

Let's remove the comment as well.

```suggestion
                <apictl.package>apictl-${apictl.version}-darwin-arm64.tar.gz</apictl.package>
```

---

Fixed with https://github.com/wso2/product-microgateway/pull/3294/commits/5ce86bde4b42be78841f82e5ad112d7eed5c9c35